src/python/nimbusml/examples/EnsembleClassifier.py

```diff
@@ -0,0 +1,46 @@
+###############################################################################
+# EnsembleClassifier
+from nimbusml import Pipeline, FileDataStream
+from nimbusml.datasets import get_dataset
+from nimbusml.feature_extraction.categorical import OneHotVectorizer
+from nimbusml.ensemble import EnsembleClassifier
+
+# data input (as a FileDataStream)
+path = get_dataset('infert').as_filepath()
+data = FileDataStream.read_csv(path)
+print(data.head())
+#   age  case education  induced  parity  ... row_num  spontaneous  ...
+# 0   26     1    0-5yrs        1       6 ...       1            2  ...
+# 1   42     1    0-5yrs        1       1 ...       2            0  ...
+# 2   39     1    0-5yrs        2       6 ...       3            0  ...
+# 3   34     1    0-5yrs        2       4 ...       4            0  ...
+# 4   35     1   6-11yrs        1       3 ...       5            1  ...
+
+
+# define the training pipeline
+pipeline = Pipeline([
+    OneHotVectorizer(columns={'edu': 'education'}),
+    EnsembleClassifier(feature=['age', 'edu', 'parity'],
```

Comment:
>EnsembleClassifier [](start = 4, length = 18)

What type of predictor is used by default?

Reply:
Multiclass Logistic Regression (LbfgsMaximumEntropyMulticlassTrainer) for multiclass, OnlineGradientDescentTrainer for regression. Right now, this is not configurable - BasePredictor parameter is not exposed in entrypoints from ML.NET side.

---
In reply to: [310263451](https://github.com/microsoft/NimbusML/pull/207#discussion_r310263451) [](ancestors = 310263451)

Reply:
could you add this ensemble documentation 


---
In reply to: [310280620](https://github.com/microsoft/NimbusML/pull/207#discussion_r310280620) [](ancestors = 310280620,310263451)